components/core/src/clp/regex_utils/constants.hpp

```diff
@@ -14,6 +36,13 @@ constexpr char cRegexStartAnchor{'^'};
 constexpr char cRegexEndAnchor{'$'};
 constexpr char cEscapeChar{'\\'};
 constexpr char cCharsetNegate{'^'};
+
+// Character bitmaps
+// This is a more complete set of meta characters than necessary, as the user might not be fully
+// knowledgeable on which meta characters to escape, and may introduce unnecessary escape sequences.
```

Comment:
I think we can just say: "The set of characters that support a preceding escape sequence."

The current wording is a bit confusing and less clear than the documentation you made.